src/applications/edu-benefits/components/ConfirmationPageContent.jsx

```diff
@@ -37,6 +38,9 @@ export function ConfirmationPageContent({
   const [isExpanded, setIsExpanded] = useState(false);
   const response = submission.response ? submission.response.attributes : {};
   const displayFormId = formId[0] === '2' ? `Form ${formId}` : formId;
+  // Prod Flag bah-23496
+  const prodFlag1990s =
+    formId === 'VRRAP' && environment.isProduction() ? 'line-height' : '';
```

Comment:
instead of this pass in a custom version of the list as the property `claimInfoListItems` with property `displayDefaultClaimList` set to false